2-collectors/scala-stream-collector/project/Dependencies.scala

```diff
@@ -51,6 +52,7 @@ object Dependencies {
     val pubsub               = "com.google.cloud"      %  "google-cloud-pubsub"    % V.pubsub
     val kafkaClients         = "org.apache.kafka"      %  "kafka-clients"          % V.kafka
     val nsqClient            = "com.snowplowanalytics" %  "nsq-java-client"        % V.nsqClient
+    val eventHub             = "com.microsoft.azure"   %   "azure-eventhubs-eph"   % V.eventHub
```

Comment:
:art: `"azure-eventhubs-eph"` is padded one character too many to the right I think